### DISCUSSION TITLE
[ FF7 ] Game Crashes after obtaining Holy Materia and descending to the bottom.

### DISCUSSION THREAD

**Discussion by JohnnyBgood2** [julianxhokaxhiu/FFNx#244] [General]
Hello,

Recently I found that my game crashes right after I obtain the Holy materia then descend to the bottom.  I hope that you can find the issue and help me to fix it immediately.

Thank you!

[Final Fantasy 7 Crash Report.zip](https://github.com/julianxhokaxhiu/FFNx/files/6464503/Final.Fantasy.7.Crash.Report.zip)

**Comment by julianxhokaxhiu** [comment]
Hi,

Thanks for creating this report but as there are no logs attached nor you're mentioning if you're using it vanilla ( FFNx only ) or through 7th Heaven or other ways, I can't help you much more than this.

If you're not using FFNx in vanilla mode, please consider approaching support channels of the relative solution you're using.

Cheers!

**Reply by JohnnyBgood2** [reply]
You're absolutely right Julian.  My apologies for the vague description.  I'm using the 7th Heaven MOD program.
I don't know how to attach a 'log' to show you what happens or what the issue would be.
But I did attach my Crash Report to my original posting.  That was the only thing I could think of to show you.

**Reply by julianxhokaxhiu** [reply]
As you're using 7th Heaven I suggest you to use its official support channels. This one is only if you FFNx in vanilla mode. Most probably is related to a mod you're using what is going on :) Cheers!

**Reply by JohnnyBgood2** [reply]
Well the thing is a window pops up in the Crash instance that specifically states to report my issue and turn in my Crash report to this website.  So that's why I did just that.  But I'll ask the mod developers on DISCORD as well to see if they know what it's talking about.  But thank you for your help!

**Reply by julianxhokaxhiu** [reply]
Welcome :)
